Update MetalLB README.md and values.yaml

cdrage opened the pull request:
Updates the MetalLB README as well as values to include an example for
Arp / BGP configuration as well as removing the ArpAddresses parameter
example that is no longer supported / used.

danderson reviewed (approved):
LGTM as author of MetalLB. I don't have commit rights on this repo though, you'll need to follow the k8s bot's instructions for how to get a review

cdrage committed:
Update MetalLB README.md and values.yaml

Updates the MetalLB README as well as values to include an example for
Arp / BGP configuration as well as removing the ArpAddresses parameter
example that is no longer supported / used.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
